### DISCUSSION TITLE
I dont know what is wrong

### DISCUSSION THREAD

**Discussion by SiKi02** [smartcontractkit/full-blockchain-solidity-course-py#1155] [Q&A] [answered]
Hi! I tried to install yarn in different ways, but the error is still the same (command not found) eventhough i have it on json doc. I also already retried to install node with homebrew and it doesn't work. I don't know if i'm doing something wrong in the process. Thanks for the help
<img width="260" alt="Captura de Pantalla 2022-02-25 a la(s) 9 19 53 p  m" src="https://user-images.githubusercontent.com/96997244/155827218-d22c6e7b-3528-410d-bf64-c736623bc73c.png">
<img width="1440" alt="Captura de Pantalla 2022-02-25 a la(s) 9 19 41 p  m" src="https://user-images.githubusercontent.com/96997244/155827219-a758bfca-01b9-4870-986f-72ad732b0daa.png">
![Captura de Pantalla 2022-02-25 a la(s) 9 19 37 p  m](https://user-images.githubusercontent.com/96997244/155827221-c19b67bb-04ae-4552-98ed-31d4681ffe50.png)
!

**Comment by n4n0b1t3** [comment]
So, OK, you don't know what is wrong! Let's start with the way you post your question here.
Please have a look at this post and update your question in a way, that others can actually help you. [Please read this first](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/980)

**Reply by SiKi02** [reply]
Thanks bro!

**Comment by cromewar** [comment, marked as answer]
Hello @SiKi02 I'm going to suppose you are using LInux/Mac, this happens when the `node_modules` are no added to your path so please do the following.

1. Please install NPM trough NVM, [Here](https://github.com/nvm-sh/nvm) is how.
2. the error persists add this commands to your  `bash_profile` or `.bashrc` file

```SHELL
export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
```

3. Be sure to install yarn using global statement:

`npm i -g yarn`

**Reply by SiKi02** [reply]
Thank you so much brother!
